pkg/client/deploymentlogs.go

```diff
@@ -34,15 +35,27 @@ func newDeploymentLogs(c *Client, namespace string) *deploymentLogs {
 
 // Get gets the deploymentlogs and return a deploymentLog request
 func (c *deploymentLogs) Get(name string, opts api.DeploymentLogOptions) *kclient.Request {
-	req := c.r.Get().Namespace(c.ns).Resource("deploymentConfigs").Name(name).SubResource("log")
+	req := c.r.Get().Namespace(c.ns).Resource("deploymentConfigs").Name(name).SubResource("log").
+		Param("follow", strconv.FormatBool(opts.Follow)).
+		Param("timestamps", strconv.FormatBool(opts.Timestamps))
```

Comment:
> How about converting the object here to a versioned and then converting to params?

It shouldn't be done in this client.  This is logically an unversioned client.  We want a method on the request like, `QueryParamsFromResource()` or something shorter for Clayton.  This particular file doesn't know which codec to use to perform the serialization.  And on the other side, you'd have to be sure the defaulting worked correctly as they were pulled out.

It's more work than I would expect for this pull.